opflexagent/utils/bridge_managers/ovs_lib.py

```diff
@@ -47,3 +51,10 @@ def ovsdb_transaction(self):
                     yield txn
                 finally:
                     self._transaction = None
+
```

Comment:
For this use case, we want it to look like it's a real bridge, so that we can exercise as much of the networking code path as possible. However, since we're faking the compute side, which is what creates these interfaces, we just create the objects in order to ensure that we get basically the same behavior. 

I'll have a look at that pull-request and do a review -- thanks!